sdk/communication/azure-communication-rooms/tests/test_rooms_client.py

```diff
@@ -225,10 +231,12 @@ def test_add_or_update_participants(self):
         for participant in update_response:
             participants.append(participant)
         assert len(participants) == 3
-        self.assertCountEqual(expected_participants, participants)
+        case = unittest.TestCase()
+        case.assertCountEqual(expected_participants, participants)
```

Comment:
I guess this isn't necessarily an issue, but it would be preferable to make these assertions without using `unittest.TestCase` if possible

Reply:
I'm trying to test that the elements of two lists are equal without considering the order. Do you have a recommendation on how I can do this without `unittest.TestCase`?

Reply:
The recommendation would be to implement a method for asserting equality, but after looking into it more, I think that what you're doing here is actually probably the best solution 🙂 I might borrow this approach for some of my own tests!